src/simulator_core/entities/assets/esdl_asset_object.py

```diff
@@ -1,11 +1,46 @@
 """Module containing classes to be able to interact with esdl objects."""
 import logging
+from dataclasses import dataclass
+
 from esdl import esdl
-from simulator_core.entities.assets import AssetAbstract, DemandCluster, ProductionCluster, Pipe
+
+from simulator_core.adapter.transforms.string_to_esdl import StringEsdlAssetMapper
 
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class EsdlKey:
+    """Class to hold the name and default value of an esdl key.
+
+    :param name: str the name of the key
+    :param default: any the default value of the key
+    """
+
+    name: str
+    default: any
+
+
+ASSET_DICT = {
+    "producer": {
+        "heating demand": EsdlKey(name="power", default=0.0),
```

Comment:
Dit voelt een beetje gek om de power van een producter heating demand te noemen. Het is ook geen heating demand maar max power.

Reply:
Het is een voorbeeld om parameters uit de component te halen. We moeten even besluiten welke parameters precies nodig zijn; ik vond het excel bestand van Ryvo niet erg duidelijk.